handlers/compute.go

```diff
@@ -123,7 +123,7 @@ func (h *ComputeHandler) InstancePull(event *revents.Event, cli *client.RancherC
 		Mode:      instancePull.Mode,
 		Complete:  instancePull.Complete,
 		Tag:       instancePull.Tag,
-		ImageUUID: instancePull.Image.Data.DockerImage.FullName,
+		ImageUUID: instancePull.Image.Data.DockerImage.Server + "/" + instancePull.Image.Data.DockerImage.FullName,
```

Comment:
Are there any scenarios where DockerImage.Server could be blank? Would it be safer to check that `DockerImage.Server != ""` before deciding to add `DockerImage.Server + "/"`?